src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingAdjustmentService.kt

```diff
@@ -290,6 +290,32 @@ class SentencingAdjustmentService(
     )
   }
 
+  @Transactional(propagation = Propagation.MANDATORY)
+  fun copyAllAdjustment(fromSentences: List<OffenderSentence>, toSentences: List<OffenderSentence>): List<SentenceIdAndAdjustments> = fromSentences.zip(toSentences).map { (fromSentence, toSentence) ->
```

Comment:
This assumes that fromSentences and toSentences match and hence can be zipped - just wondering if that assume is valid?